### PR TITLE
Added more Arabic translation

### LINKED ISSUE #7645: This sentences has not been translated into Arabic
- There are many sentences that have not been translated into Arabic, such as:
https://github.com/files-community/Files/blob/5580cc7c7c9d431dcafd2dedf940258dfe3d1503/src/Files/Strings/ar/Resources.resw#L1240
https://github.com/files-community/Files/blob/5580cc7c7c9d431dcafd2dedf940258dfe3d1503/src/Files/Strings/ar/Resources.resw#L964
https://github.com/files-community/Files/blob/5580cc7c7c9d431dcafd2dedf940258dfe3d1503/src/Files/Strings/ar/Resources.resw#L856
https://github.com/files-community/Files/blob/5580cc7c7c9d431dcafd2dedf940258dfe3d1503/src/Files/Strings/ar/Resources.resw#L664

- The translation of a byte into Arabic is supposed to be "بايت", not the letter B.
https://github.com/files-community/Files/blob/5580cc7c7c9d431dcafd2dedf940258dfe3d1503/src/Files/Strings/ar/Resources.resw#L421

### PR DESCRIPTION
**Pull request by 0x61nas**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #7645
- Related #7645 

**Details of Changes**
Add details of changes here.
- Added more Arabic translations 
- Fixed some Arabic translations 

**Validation**
How did you test these changes?
- [ ] Built and ran the app
- [ ] Tested the changes for accessibility

### CODE REVIEW

**Comment by Anas-Elgarhy** [comment]
I am still working on translating more texts, but I made a withdrawal request in order to facilitate the process of reviewing changes before they become too big and difficult to review.

**Comment by yaichenbaum** [comment]
@Anas-Elgarhy before we merge this, can you add your agreement to #7625?

**Comment by 0x61nas** [comment]
@yair100  I've already added it

https://github.com/files-community/Files/issues/7625#issuecomment-1013118605

File: src/Files/MultilingualResources/Files.ar.xlf [hunk 1]
```diff
@@ -1576,31 +1576,31 @@
         </trans-unit>
         <trans-unit id="SidebarCloudDrives" translate="yes" xml:space="preserve">
           <source>Cloud Drives</source>
-          <target state="new">Cloud Drives</target>
+          <target state="needs-review-translation">محركات السحابة</target>
```
Comment (yaichenbaum) [inline src/Files/MultilingualResources/Files.ar.xlf:1579]: If the translations are good, can you change the state to `translated`?

Comment (Anas-Elgarhy) [reply src/Files/MultilingualResources/Files.ar.xlf:1579]: I thought it was inappropriate but after thinking I think it is, at least for the time being, so I will change the state now, and I may reconsider some of the other strings

Comment (Anas-Elgarhy) [reply src/Files/MultilingualResources/Files.ar.xlf:1579]: I fixed the issue, and fixed some other bugs, thanks <3

File: src/Files/MultilingualResources/Files.ar.xlf [hunk 2]
```diff
@@ -1576,379 +1576,380 @@
...
         </trans-unit>
         <trans-unit id="DriveTypeFloppyDisk" translate="yes" xml:space="preserve">
           <source>Floppy Disk Drive</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">محرك الأقراص المرنة</target>
+          <target state="translated" state-qualifier="tm-suggestion"> محرك الأقراص المرنة (Floppy Disk)</target>
         </trans-unit>
         <trans-unit id="DriveTypeNetwork" translate="yes" xml:space="preserve">
           <source>Network Drive</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">محرك أقراص الشبكة</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">محرك الشبكة</target>
```
Comment (yaichenbaum) [inline src/Files/MultilingualResources/Files.ar.xlf:1679]: Can you update the state to translated?

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**